### ISSUE TITLE
Add unit tests

### ISSUE THREAD

**Issue by Ohiekkar** [mbuet2ner/JEasyCrypto#21]
We should definitely add some unit tests. How else can we be sure that the different encrypting methods produce correct results?

**Comment by mbuet2ner** [comment]
I agree. This would also allow us to leverage continuous integration e.g. [Travis CI](https://travis-ci.org/). Would be happy if someone would add this.

**Comment by pottumuusi** [comment]
Have never written test code for crypto implementations, but maybe these provide some inspiration.

#### Android crypto tests
https://android.googlesource.com/platform/external/bouncycastle/+/397d32894b89b506dc318e0f83446187c9b76ebe/bcprov/src/main/java/org/bouncycastle/crypto/test

#### Project Wycheproof tests crypto libraries against known attacks.
https://github.com/google/wycheproof
https://www.schneier.com/blog/archives/2016/12/google_releases.html

There was rot13 mentioned in sources. Maybe it is tested somewhat like encoding is?
https://github.com/openssl/openssl/blob/master/test/asn1_encode_test.c
